doc/api/process.md

```diff
@@ -1336,6 +1336,45 @@ In custom builds from non-release versions of the source tree, only the
 `name` property may be present. The additional properties should not be
 relied upon to exist.
 
+## process.resourceUsage()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Object} containing resource usage measures for the current process
+
+Note: Windows only provides the following resource usage measures:
+
+- `userCpuTimeUsedSec`
+- `userCpuTimeUsedMs`
+- `systemCpuTimeUsedSec`
+- `systemCpuTimeUsedMs`
```

Comment:
It looks like it was added in libuv v1.10.0. If/when this PR gets backported, we'd need to have a separate doc change because v4.x and v6.x have older versions of libuv which may/may not be upgraded.